### DISCUSSION TITLE
/gridpoints/{wfo}/{x}/{y}/stations not honoring limit parameter

### DISCUSSION THREAD

**Discussion by netbymatt** [weather-gov/api#606] [General]
When querying the above endpoint with a limit parameter set I get what appears to be all the stations near the grid point, typically >50 even if I set the limit parameter smaller such as:
https://api.weather.gov/gridpoints/MPX/108,31/stations?limit=20
The problem does not appear to be specific to a WFO.

**Comment by dwhitemv25** [comment]


**Reply by JJKraw** [reply]
<img width="679" alt="Screenshot 2022-12-14 at 19 38 52" src="https://user-images.githubusercontent.com/67399409/207745339-0f82fe35-e129-40c0-a4af-7e67d3314398.png">

Sure looks like limit is documented as a query parameter.

**Reply by netbymatt** [reply]
It looks like the url in my post and the screenshot you provided are the same endpoint. Am I missing something?

**Reply by JJKraw** [reply]
I was replying to dwhitemv25, who said the endpoint didn’t support limit. Just showing that the documentation says that it does support limit.

**Reply by dwhitemv25** [reply]
Well never mind. I am blind. Sorry for the wrong information.
